### DISCUSSION TITLE
Can't open canvas, it crashes when I try to open

### DISCUSSION THREAD

**Discussion by stepsteph** [thatskymod/Sky-CotL-Scripts#56] [Q&A] [answered]
I can't open canvas, it crashes when I try to open it and doesn't want to work. Is there any update or fix for canvas? Because when I try to open something appear saying: "Sorry we can't open because that app have a bug, wait for any update ".

Even some of my friends can't open canvas. Please is there's any update, let me know.

**Comment by thatskymod** [comment]
1. **Simple Things First**:
   Let's try to isolate the issue by launching the original Sky without Canvas. Note that there is an ongoing issue where some Android users are experiencing crashes with the new Sky update 0.24.0[^1] regardless of using mods. This is a known bug within Sky itself and is not specific to Canvas:

   > ### Known Issues Update 0.24.0
   >
   > 🔹 We are aware of the crashing issues occurring on some Android devices and are currently investigating the cause.
   >
   > We appreciate your patience and apologize for any inconvenience this may cause as we work to resolve this issue.

2. If the issue is specific to Canvas, could you please provide a screenshot of your Mod Manager or list the mods you have installed? This information will aid us in finding the root cause.

Thank you for your patience in the meantime.

[^1]: [Known Issue announcement (Discord)](https://discord.com/channels/575762611111592007/628684058414678026/1195524423982268577)

**Reply by SKazehaya** [reply]
I'm having the same problem, andit doesn't seem to be from the original Sky game. 

Here's a video showing that canva doesn't open awhile the og does:

https://github.com/thatskymod/Sky-CotL-Scripts/assets/154921570/c30b01ed-387a-42c5-b1fb-d740336fa5f0

This is a screenshot of the mod manager list:

![Screenshot_2024-01-13-21-33-44-962_git artdeell skymodloader](https://github.com/thatskymod/Sky-CotL-Scripts/assets/154921570/0775d06c-d45d-4b3c-9bab-8fdfc0c26c0b)

Thanks in advance.

**Reply by stepsteph** [reply]
This are the mods that I have installed:
![Screenshot_20240113_172902_Canvas](https://github.com/thatskymod/Sky-CotL-Scripts/assets/146548916/e5f97bfc-4a29-42bd-ae9e-830c6393129c)
smite

**Reply by stepsteph** [reply]
The same thing happens to me, I can't open original sky but not canvas

**Comment by SKazehaya** [comment, marked as answer]
Ok, so i uninstalled all libs and the problem seems to be libsmite, cuz without it, all the other mods work!

*I tested them one by one, and when I added it back, canvas stopped opening


https://github.com/thatskymod/Sky-CotL-Scripts/assets/154921570/9180ba37-3def-45c3-acf8-57e19590a71c

![Screenshot_2024-01-13-21-54-19-698_git artdeell skymodloader-edit](https://github.com/thatskymod/Sky-CotL-Scripts/assets/154921570/37f20ed4-bd6e-49f3-99b8-b65fd753b2f8)

**Comment by thatskymod** [comment]
I was able to confirm that the problem indeed stems from **libsmite-1.0.7d.so**.

The lib has thus been removed from the repository and will be added back once a working update is available.

Sorry for any inconvenience this may have caused, and thank you for your patience.

**Reply by getosotoruoo** [reply]
Well my sMite ain't work anymore is it only me? I tried to deleted it and re-download it but that was not worked

**Reply by bdalrhmnghwyly79** [reply]
> I was able to confirm that the problem indeed stems from **libsmite-1.0.7d.so**.
> 
> 
> 
> The lib has thus been removed from the repository and will be added back once a working update is available.
> 
> 
> 
> Sorry for any inconvenience this may have caused, and thank you for your patience.

**Comment by TokuhasDoku** [comment]
does anyone know if i can run canvas through an emulator i don't have an android

**Comment by potato1231205** [comment]
When I start Canvas and press Live, the app runs until halfway through, but then it crashes and I can't play the game. If anyone knows how to solve this, please let me know.
